src/kernels/dynamic_igemm/igemm_bwd_gtc_dynamic.s

```diff
@@ -1227,6 +1227,8 @@ L_igemm_v4r1_bwd_dynamic_end:
     .amdhsa_next_free_sgpr 72
     .amdhsa_ieee_mode 0
     .amdhsa_dx10_clamp 0
+    //xnack disabled by default for asm kernels
+    .amdhsa_reserve_xnack_mask 0
```

Comment:
[Future] Note that documentation says that the default of the `.amdhsa_reserve_xnack_mask` is Target Feature Specific, and, most likely, the changes like this are NOT required. Let's keep this change, however, for the sake of reliability. I think we will roll back such changes after upcoming release.

Reply:
directive ``.amdhsa_reserve_xnack_mask``  added for each asm kernel at the request of management.

Reply:
"management" is an abstract notion. Who?